libsplinter/src/registry/mod.rs

```diff
@@ -24,6 +24,8 @@
 //! [`RegistryWriter`]: trait.RegistryWriter.html
 //! [`RwRegistry`]: trait.RwRegistry.html
 
+#[cfg(feature = "registry")]
```

Comment:
This should be changed to "registry-client".

Reply:
Fixed